### ISSUE TITLE
Unable to follow tutorial

### ISSUE THREAD

**Issue by ayertay** [karlpauwels/simtrack#34]
Hello,

I'm a newbie in ROS. I'm having trouble following the tutorial. I'm using kinect v1, and after I run roslaunch simtrack_nodes camera_kinect.launch, am I supposed to open another terminal and run rviz?

If I run rviz, click add and add by topic, should /simtrack/image be there? because I only have /camera/stuff, and no /simtrack.

After I do roslaunch, I get: 

```
[ WARN] [1511518317.829956847]: Could not find any compatible depth output mode for 1. Falling back to default depth output mode 1.
[ INFO] [1511518317.836755515]: rgb_frame_id = 'camera_rgb_optical_frame' 
[ INFO] [1511518317.836794898]: depth_frame_id = 'camera_depth_optical_frame' 
[ WARN] [1511518317.846440662]: Camera calibration file /home/ali/.ros/camera_info/depth_A00363A06639127A.yaml not found.
[ WARN] [1511518317.846480745]: Using default parameters for IR camera calibration.
[ INFO] [1511518320.530530592]: Stopping device RGB and Depth stream flush.
```

Which I'm thinking is normal, isn't it? 

I installed OpenCV 2.4.13.4, libfreenect, openni, openni2, libusb, camera_manager, freenect, SensorKinect, CUDA 7.5 since I'm on Indigo.

Also, when I tried to just add TF in rviz like in http://www.karlpauwels.com/wp-content/uploads/2014/11/pose_estimation_rviz.jpg I didn't have 'kinect' showing under frames section.  
I'm sorry if this question is too basic.

Thank you very much!

**Comment by karlpauwels** [comment]
Hi,

After launching the camera, you should also launch simtrack itself (in a new terminal):
````
roslaunch simtrack_nodes multi-rigid.launch
````
Alternatively, you can modify the first line in [``main.launch``](https://github.com/karlpauwels/simtrack/blob/master/simtrack_nodes/launch/main.launch) to point to ``camera_kinect.launch``. That way you can launch both nodes together using:
````
roslaunch simtrack_nodes main.launch
````
Concerning the frames, you'll be looking for the frames shown in your output: ``camera_rgb_optical_frame`` and ``camera_depth_optical_frame``.

Hope this helps.

**Comment by ayertay** [comment]
I ran camera_kinect.launch, and multi-rigid.launch. From terminal I ran camera_kinect I get same output:

`[ WARN] [1512146085.444890728]: Could not find any compatible depth output mode for 1. Falling back to default depth output mode 1.
[ INFO] [1512146085.455627289]: rgb_frame_id = 'camera_rgb_optical_frame' 
[ INFO] [1512146085.455686602]: depth_frame_id = 'camera_depth_optical_frame' 
[ WARN] [1512146085.465538471]: Camera calibration file /home/ali/.ros/camera_info/depth_A00363A06639127A.yaml not found.
[ WARN] [1512146085.465678181]: Using default parameters for IR camera calibration.
[ INFO] [1512146088.244654091]: Stopping device RGB and Depth stream flush.
`

From the second terminal where I ran multi-rigid.launch I get:

`PARAMETERS
 * /rosdistro: indigo
 * /rosversion: 1.11.21
 * /simtrack/detector/device_id: 0
 * /simtrack/detector/num_iter_ransac: 1000
 * /simtrack/detector/vec_size: 4
 * /simtrack/model_names: ['ros_fuerte', 'r...
 * /simtrack/model_path: /home/ali/my-ws/s...
 * /simtrack/optical_flow/cons_thres: 0.5
 * /simtrack/optical_flow/consistent: True
 * /simtrack/optical_flow/four_orientations: False
 * /simtrack/optical_flow/median_filter: True
 * /simtrack/optical_flow/n_scales: 0
 * /simtrack/tracker/color_only_mode: False
 * /simtrack/tracker/device_id: 0
 * /simtrack/tracker/far_plane: 10.0
 * /simtrack/tracker/key_bits: 4
 * /simtrack/tracker/max_proportion_projected_bounding_box: 10.0
 * /simtrack/tracker/max_samples: 500000
 * /simtrack/tracker/max_t_update_norm_squared: 0.04
 * /simtrack/tracker/n_icp_inner_it: 3
 * /simtrack/tracker/n_icp_outer_it: 3
 * /simtrack/tracker/near_plane: 0.001
 * /simtrack/tracker/reliability_threshold: 0.15
 * /simtrack/tracker/sparse_intro_allowed_reliability_decrease: 0.02
 * /simtrack/tracker/sparse_intro_reliability_threshold: 0.3
 * /simtrack/tracker/w_ar_flow: 1.0
 * /simtrack/tracker/w_disp: 1.0
 * /simtrack/tracker/w_flow: 1.0
 * /simtrack/use_compressed_streams: False

NODES
  /
    simtrack (simtrack_nodes/multi_rigid_node)

ROS_MASTER_URI=http://localhost:11311

core service [/rosout] found
process[simtrack-1]: started with pid [11531]
`

Should there be something starting? Is this normal? It doesn't give any errors, but also doesn't give anything else.

Thanks in advance!

**Comment by ayertay** [comment]
Thank you very much! I'll type this so that if others run into the same problem, they can follow.

I found the file main.launch, opened it, and changed xtion to camera_kinect. Then run:
`roslaunch simtrack_nodes main.launch`

It gave this:

`[ WARN] [1511518317.846480745]: Using default parameters for IR camera calibration.
[ INFO] [1511518320.530530592]: Stopping device RGB and Depth stream flush.`

Then I opened a new terminal and run rviz:

`rviz rviz`

There in Add->Add by topic I chose simtrack/image, that showed the output from camera. I printed out the models (SIFT) and showed to camera. It identified it. Also following openni kinect tutorial helped.
